src/black/mode.py

```diff
@@ -30,77 +29,45 @@ class TargetVersion(Enum):
     PY39 = 9
     PY310 = 10
 
-    def is_python2(self) -> bool:
-        return self is TargetVersion.PY27
-
 
 class Feature(Enum):
```

Comment:
Unrelated, but should we use `enum.auto()` to avoid numbering features ourselves?

Reply:
I have no preference, I do like the grouping nature of the current numbering but the empty lines do a good enough job at delimiting groups as well.

Reply:
Up to you I think. On one hand, changing one value like `unicode literal` here leads to the need to change many others as well, but on the other, there shouldn't be any changes like this on the horizon and the grouping *is* nice, albeit only really impactful on these few lines unless the values are actually used somewhere, which I doubt. *Also* not strictly in the scope of this PR either 😄 but I like the boy scout rule.

Reply:
What you have is fine but I'd just remove what needs removing and not renumber anything.

Reply:
Fair enough, I'll undo the renumbering to preserve git blame.